docs/en/middleware.rst

```diff
@@ -291,5 +291,6 @@ This is, because we extend our handler based on the RedirectHandler present in t
     
 The ``custom_param`` appears in the ``$options`` array given to you in the ``handle()`` function inside your ``CustomRedirectHandler`` if you wish to add some more config parameters to your functionality.
 
-You can look at `CakeRedirectHandler <https://github.com/cakephp/authorization/blob/2.next/src/Middleware/UnauthorizedHandler/CakeRedirectHandler.php>`__ or `RedirectHandler <https://github.com/cakephp/authorization/blob/2.next/src/Middleware/UnauthorizedHandler/RedirectHandler.php>`__ 
+You can look at `CakeRedirectHandler <https://github.com/cakephp/authorization/blob/2.next/src/Middleware/UnauthorizedHandler/CakeRedirectHandler.php>`
```

Comment:
Doesn't this need a `_` still?

Reply:
In fact, yes.
My Geany editor doesn't display undescores correctly, so I misunderstood them as trailing whitespaces. I revert them in english and add them in french.